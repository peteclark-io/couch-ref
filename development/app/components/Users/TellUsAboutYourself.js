'use strict';

import React from 'react';
import {connect} from 'react-redux';

import styles from './styles.css';

const TellUsAboutYourself = React.createClass({

  render: function() {
    return (
      <div>
         <h1 className={styles.brand}>Tell Us About Yourself!</h1>
         {this.props.children}
      </div>
    );
  }
});

export default TellUsAboutYourself;
