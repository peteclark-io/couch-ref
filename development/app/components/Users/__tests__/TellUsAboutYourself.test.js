'use strict';

import React from 'react';
import TellUsAboutYourself from '../TellUsAboutYourself';
import renderer from 'react-test-renderer';
import {mount, shallow} from 'enzyme';

it('Renders.', () => {
   const rendered = renderer.create(
      <TellUsAboutYourself />
   );
   expect(rendered.toJSON()).toMatchSnapshot();
});

it('Renders children.', () => {
   const rendered = renderer.create(
      <TellUsAboutYourself children={<h1>hi</h1>} />
   );
   expect(rendered.toJSON()).toMatchSnapshot();
});
