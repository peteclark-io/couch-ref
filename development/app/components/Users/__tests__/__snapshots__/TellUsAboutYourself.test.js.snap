exports[`test Renders children. 1`] = `
<div>
  <h1
    className="brand">
    Tell Us About Yourself!
  </h1>
  <h1>
    hi
  </h1>
</div>
`;

exports[`test Renders. 1`] = `
<div>
  <h1
    className="brand">
    Tell Us About Yourself!
  </h1>
</div>
`;
